app/_sheet/page.tsx

```
@@ -93,16 +94,16 @@ export default function Sheet() {
+  function handleDragEnd(event: DragEndEvent) {
     const { active, over } = event;
 
-    if (active.id !== over.id) {
+    if (active.id !== over?.id) {
       setItems((items) => {
         const oldIndex = items[0].items.findIndex(
           (item) => item.id === active.id
         );
         const newIndex = items[0].items.findIndex(
-          (item) => item.id === over.id
+          (item) => item.id === over!.id
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)